operations/utility_storage/Troubleshoot_Ceph_Container_Image_Isses.md

```diff
@@ -0,0 +1,23 @@
+# Ceph Container Image Issues
+
+## Scope
+
+A general reference page for troubleshooting issues where Ceph cannot pull an image or a service cannot start due to a missing image.
+
+## Scenario 1
+
+**Description:**
+
+Ceph service will not start due to missing image.
+
+**Example error message:**
+
+```text
+   insert text
```

Comment:
is this content coming in another PR?

Reply:
lol..  you got me..  I forgot I started this page.  let me yank it out.

Reply:
removed